### PR TITLE
feat(Consents): Terms should be editable from Django admin view.

### PR DESCRIPTION
**Pull request by lauryndbrown**
Fixes https://github.com/carpentries/amy/issues/1831.

Dependent upon: https://github.com/carpentries/amy/pull/1832

### CODE REVIEW

**Comment by lauryndbrown** [comment]
Current state:
Screen Shot 2021-03-12 at 9 51 20 AM
Screen Shot 2021-03-12 at 9 51 28 AM
Screen Shot 2021-03-12 at 9 51 44 AM

**Comment by lauryndbrown** [comment]
Screen Shot 2021-03-14 at 12 41 36 PM

File: amy/consents/migrations/0002_auto_20210314_1928.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.17 on 2021-03-14 19:28
```
Comment (pbanaszkiewicz) [inline amy/consents/migrations/0002_auto_20210314_1928.py:1]: Please introduce the changes to `0001` migration.

**Review by pbanaszkiewicz — CHANGES_REQUESTED** [review]
Looks good! Can you introduce the `blank` option to migration `0001` instead of creating a new migration?

**Comment by lauryndbrown** [comment]
@pbanaszkiewicz could you take another look?

**Review by pbanaszkiewicz — APPROVED** [review]
Looks good, thanks for the changes.

**Pull request merged**